### DISCUSSION TITLE
Questions about Apple chip hardware acceleration

### DISCUSSION THREAD

**Discussion by MrChenhtlss** [rustdesk/rustdesk#7603] [Q&A] [answered]
I'm using Apple's M1 chip but I found that it doesn't support H265 or H265 encoding.

**Comment by 21pages** [comment, marked as answer]
It's in plan.

**Comment by MrChenhtlss** [comment]
Okay, this is very cool.

**Comment by MrChenhtlss** [comment]
I can know what models of Intel platform can support hardware acceleration

**Reply by a15355447898a** [reply]
I was wondering the same thing, the hardware encoder isn't recognized on my i5-8500.
